docs/ZAP-on-Windows.md

```diff
@@ -0,0 +1,79 @@
+# Install ZAP from Source on Windows
+
+## Windows-specific Prerequisite Settings
+
+**1. Windows Powershell:** 
+
+In desktop search bar, input "Windows Powershell" and run as administrator. Run all the following commands inside Powershell.
+
+**2. Chocolatey:**
+
+Install from https://chocolatey.org/install
+
+Check if installed properly:
+```
+choco -v
+```
+Install pkgconfiglite package:
+```
+choco install pkgconfiglite
+```
+
+**3. Install Node if you have not:**
+```
+choco install nodejs-lts
+```
+*the version has to be 18 to pass version check test, after install, check with "node -v"
+
+*if you install Node already, and fail some tests similar to "cannot find Node", reinstall Node with chocolatey again
+
+**4. Follow the instruction page for other installation details**
+
+https://github.com/project-chip/zap/blob/master/docs/development-instructions.md
+
+## Fix Possible Error
+
+**1. sqlite3**
+
+When running the ZAP app, if has error about "sqlite3.node" occured in a pop out window, run:
+```
+npm rebuild sqlite3
+```
+
+**2. electron-builder:**
+
+When doing npm install, in post-install, if has error on the following command, related to "electron-builder install-app-deps", "node_modules/canvas" or "node-pre-gyp", it is because the electron-builder is not completely compatible with Windows and the installation error will not cause failure in the ZAP app
```

Comment:
Currently we do not have easy solutions for this. electron-builder and canvas have some version issue and are not completely compatible on Windows. Since the error only affects the release package of Windows, the users might not need to fix this in their scope?